ui/src/features/Editor/index.tsx

```diff
@@ -135,11 +134,14 @@ export default function Editor({
             project={currentProject}
             yDoc={yDoc}
           />
-          <ParamsPanel
-            selected={locallyLockedNode}
-            onDataSubmit={handleNodeDataUpdate}
-            onWorkflowRename={handleWorkflowRename}
-          />
+          {openNode && (
```

Comment:
Since we are relying on ParamsPanel's dialog to open/close it, we shouldn't need to conditionally render the ParamsPanel here with openNode &&